INSTALL-macos.md

```diff
@@ -0,0 +1,89 @@
+# Installing the CircleCI Runner on macOS
+
+## Create a runner configuration
+Choose a user to run the CircleCI agent as. These instructions will refer to the selected user as `USERNAME`.
+
+Complete the below template, with the various capitalised parameters filled in, and save as `launch-agent-config.yaml`.
+```yaml
+api:
+    auth_token: AUTH_TOKEN
+runner:
+    name: NAME
```

Comment:
is there an invalid YAML value we can use here to ensure the customer doesn't just copy NAME itact?

Reply:
You could maybe put a bad anchor in, but that would probably cause more confusion than help

Reply:
We could also consider making `name` optional?

Reply:
we could add validation to disallow `NAME` as a value 😁

Reply:
That's not a bad idea!